scripts/idl/README.md

```diff
@@ -94,6 +94,10 @@ server cluster AccessControl = 31 {
   // These defaults can be modified to any of view/operate/manage/administer roles.
   attribute access(read: manage, write: administer) int32u customAcl = 3;
 
+  // Attributes may be fabric-scoped as well by tagging them as `fabric`.
+  fabric readonly attribute int16u myFabricAttr = 22;
```

Comment:
This is wrong.  Attributes cannot be fabric-scoped at all...

Reply:
Offline conversation results:
  - fabric scoped on attributes is only for "lists of structures"
  - the structures themselves will be tagged as "fabric scoped" as this pulls in a fabric index attribute into them
  
  So followup changes should be:
     - remove fabric scoped from attributes
     - tag structs as fabric scoped (assuming zap knows about this)